### PR TITLE
Added-DS/Radix Sort Algorithm

### PR DESCRIPTION
**Pull request by Meghag5**
# Description

Radix sort algorithm is implemented.

#7 

## Type of change

Please check options that are relevant to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New Data Structure 
- [x] New Algorithm

# Checklist:

- [x] I have squashed my commits
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My code is a working bug-free code

### CODE REVIEW

**Comment by ay2306** [comment]
You have again submitted the hidden files, please delete them and add `.DS_Store` to `.gitignore`

File: Algorithms/Sorting/RadixSort.cpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    for(int i=1;i<range;i++){
+        count[i]=count[i]+count[i-1];
+    }
+    // placing input array elements into output array in proper
+    //  positions such that the result is a sorted array
+
+    for(int i=n-1;i>=0;i--){
+        output[count[(arr[i]/div)%10]-1]=arr[i];
+        count[(arr[i]/div)%10]--;
+    }
```
Comment (ay2306) [inline Algorithms/Sorting/RadixSort.cpp:35]: You need to check this part.

File: Algorithms/Sorting/RadixSort.cpp [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        arr[i]=output[i];
+    }
+}
+void radixSort(int *arr,int n){
+    int m=maxValue(arr,n);
+    int div;
+    for(int div=1;m/div>0;div*=10){
+        countingSort(arr,n,div);
+    }
+}
```
Comment (ay2306) [inline Algorithms/Sorting/RadixSort.cpp:48]: Radix sort is not just simply running counting sort `log(A)` times. On top of that, int div at line 44 is unnecessary.

**Review by ay2306 — CHANGES_REQUESTED** [review]
The code seems logically flawed. However, I have just commented on the first read basis. Please check your code or if you think your code is correct tell me and I will test it as I have my suspicion on this code to be logically wrong.

**Comment by Meghag5** [comment]
@ay2306 pls check my code again. It's output is correct

**Comment by Meghag5** [comment]
@ay2306 all the changes are made pls check

**Comment by ay2306** [comment]
@Meghag5 Hey, sorry I had some doubts understanding the implementation this way. But after spending time on it, code makes sense. Merging your code.

**Pull request merged**